topic0KeyPoint1/src/oop3_4/MyTime.java

```diff
@@ -79,17 +81,14 @@ public void setSecond(int second) throws IllegalArgumentException {
 	}
 
 	public static boolean isValidHour(int hour) {
-		if (hour >= 0 && hour <= 23)
-			return true;
-		else
-			return false;
+
+		return (hour >= 0 && hour <= 23) ? true : false;
```

Comment:
Good, but this still is an IF. You are doing this:
if some expression evaluates true, then return true; else return false

That's redundant, you can avoid the if and return the result of the expression :)